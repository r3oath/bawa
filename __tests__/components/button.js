import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../../components/button';

describe('Button', () => {
  it('Matches previous snapshot', () => {
    const component = renderer.create(<Button />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
