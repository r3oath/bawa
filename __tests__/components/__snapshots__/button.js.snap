// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Matches previous snapshot 1`] = `
<button
  className="button-l4um5n-0 cftsCa"
/>
`;
